Gemfile

```diff
@@ -61,8 +61,6 @@ gem "waste_carriers_engine",
 # manage feature toggle (create / update / delete) from the back-office.
 gem "defra_ruby_features"
 
-gem "defra_ruby_govpay"
-
```

Comment:
Is defra_ruby_govpay required by waste-carriers-engine now?

Reply:
Yes - it always was in the gemspec, so not needed here